src/libraries/Common/src/Interop/Interop.Ldap.cs

```diff
@@ -182,14 +182,22 @@ internal struct LDAP_TIMEVAL
     [StructLayout(LayoutKind.Sequential)]
     internal sealed class BerVal
     {
+#if TARGET_UNIX
+        public nint bv_len;
+#else
         public int bv_len;
```

Comment:
You can use the CLong type to represent this type shape without a `#if`.

Reply:
Thanks - did that; removed the changes to the csproj.